extensions/amp-story/0.1/amp-story.css

```diff
@@ -115,6 +114,10 @@ amp-story[standalone]:fullscreen {
   background-color: rgba(0, 0, 0, 0.2) !important;
 }
 
+.i-amphtml-story-system-layer .i-amphtml-story-button {
+  float: left;
```

Comment:
`!important` any styles that shouldn't be overridden; if they are allowed to be overridden, move them to the separate CSS file.

Reply:
Done.